src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,15 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../api/types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC <Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} />
```

Comment:
don't forget to add key by mapping the array
```suggestion
      <TodoInfo key={todo.id} todo={todo} />
```